RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+Release Notes
+
+## OAF Version 1.5.4
```

Comment:
We've been including the mifos version that the channel connector is based on. For example version `1.1.1-mifos-1.5.3` means it's OAF version `1.1.1` based on mifos version `1.5.3`

You can check [the existing tags](https://github.com/one-acre-fund/ph-ee-connector-channel/tags) to see what we've been using as versions so far

Additionally, in azure-pipelines.yaml, you can make a few changes to always retrieve the version from build.gradle instead of relying on tags. I wanted that to be done during umuganda. But if you have the time, you can go ahead and make the change

Reply:
@victorjbassey Thanks for the review.. Change the version to 1.1.2-mifos-1.5.3 and refactored the image tag generation in azure-pipleine